lms/envs/common.py

```diff
@@ -404,6 +404,9 @@
 
     # Whether to display the account deletion section the account settings page
     'ENABLE_ACCOUNT_DELETION': True,
+
+    # Set to enable Journal integration
+    'ENABLE_JOURNAL_INTEGRATION': True,
```

Comment:
Which reminds me of something I forgot to add: Can we change this to be a Waffle switch instead so we can toggle this setting at runtime - rather than depend on code deployment?